cmd/tstr/ctl_test_list_cmd.go

```diff
@@ -29,5 +30,7 @@ var ctlTestListCmd = &cobra.Command{
 }
 
 func init() {
+	ctlTestListCmd.PersistentFlags().StringVar(&ctlTestNamespace, "namespace", "", "The namespace to use.")
+	ctlTestListCmd.MarkPersistentFlagRequired("namespace")
```

Comment:
Good catch, I think this could be moved to replace https://github.com/nanzhong/tstr/blob/main/cmd/tstr/ctl_test_cmd.go#L15.

Reply:
added them in ctl_test_cmd
https://github.com/nanzhong/tstr/pull/116/commits/65d2e3b027c2136d56c1134617d8b5331771771c